styles/css/CarePackages/package-request.scss

```diff
@@ -0,0 +1,4 @@
+.lbh-button.package-request-button {
+  width: 194px;
```

Comment:
probably better to set this as min-width instead of width, and set it to 200px (196px is a random amount)

Reply:
I will change width to min-width. 196px is not a random amount, i take it from figma

Reply:
The width there is 194 not 196.